meter.go

```diff
@@ -227,7 +258,7 @@ func (ma *meterArbiter) tick() {
 func (ma *meterArbiter) tickMeters() {
 	ma.RLock()
 	defer ma.RUnlock()
-	for _, meter := range ma.meters {
+	for meter := range ma.meters {
```

Comment:
~doesn't this syntax only work in newer Go? I would leave it as is to avoid breaking bw compatibility.~

Edit: I see tests pass on all the old versions, disregard.